gluten-core/src/main/scala/org/apache/gluten/execution/BasicScanExecTransformer.scala

```diff
@@ -92,7 +101,12 @@ trait BasicScanExecTransformer extends LeafTransformSupport with BaseDataSource
     }
 
     val validationResult = BackendsApiManager.getSettings
-      .supportFileFormatRead(fileFormat, fields, getPartitionSchema.nonEmpty, getInputFilePaths)
+      .supportFileFormatRead(
```

Comment:
`supportFileFormatRead` is not only checking file format now. Better to change it to other name, like "validateScan" or some other a bit common name.

Reply:
Updated, thanks